test/fw/ptl/utils/pbs_dshutils.py

```diff
@@ -1300,7 +1297,8 @@ def is_localhost(self, host=None):
             self._h2l[host] = True
             return True
         # on a shasta machine, the name returned by `hostname` (pbs-host) is
-        # different than the one we tell PTL to use (pbs-service-nmn).
+        # different than the one we tell PTL to use (pbs-service-nmn). This
```

Comment:
Just for my understanding, can you please explain why we ask PTL to use a different "local" hostname than the one that `hostname` returns ? Or if this is documented, point me to it? Thanks.